code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -371,9 +381,37 @@ static HookFunction initFunction([]()
 		auto location = hook::get_pattern<char>("F6 83 ? ? ? ? 07 75 ? 44 0F");
 
 		CurrentRPMOffset = *(uint32_t*)(location - 42);
+		ClutchOffset = CurrentRPMOffset + 12;
 		ThrottleOffsetOffset = CurrentRPMOffset + 16;
 	}
 
+	{
+		auto location = hook::get_pattern<char>("45 0f 57 ? f3 0f 11 ? ? ? 00 00 f3 0f 5c");
+
+		WheelSuspensionCompressionOffset = *(uint32_t*)(location + 8);
+		WheelRotationSpeedOffset = *(uint32_t*)(location + 8) + 0xC;
+	}
+
+	{
+		char* location;
+		if (Is2060()) {
+			location = hook::get_pattern<char>("0F 2F ? ? ? 00 00 0F 97 C0 EB ? D1");
```

Comment:
What about `0F 2F 8B ? ? 00 00 0F 92 C0`? Seems valid for 1604 *and* 2060.

Reply:
I noticed that, but ikt had the two separated in his code so I figured he would have had a reason for doing that. He certainly knows a lot more than I do.

Reply:
I tested the reduced pattern and it doesn't work. That extra byte is necessary.